report/renderer/details-renderer.js

```diff
@@ -376,7 +379,106 @@ export class DetailsRenderer {
   }
 
   /**
-   * @param {{headings: TableColumnHeading[], items: TableItem[]}} details
+   * Adorn a table row element with entity chips based on [data-entity] attribute.
+   * @param {HTMLTableRowElement} rowEl
+   */
+  _adornTableRowWithEntityChips(rowEl) {
```

Comment:
nit: `chip` as a term only comes up in this function. Possible to drop extra terminology? just `_adornEntityTableRow`?